src/store/ConfigStore.ts

```diff
@@ -20,10 +23,25 @@ declare const window;
 const webRoot = (window as any).WEB_ROOT ? (window as any).WEB_ROOT : undefined;
 const persistKey = 'kiali-' + (webRoot && webRoot !== '/' ? webRoot.substring(1) : 'root');
 
+// Needed to be able to whitelist fields but allowing to keep an initialState
+const whitelistInputWithInitialState = (reducerName: string, inboundPaths: string[], initialState: any) =>
+  createTransform(
```

Comment:
I have not heard of this `createTransform` before. I would have just used `localStorage` Api. But, this works. +1

Reply:
I was going to do that, started to read how to tackle the problem that way. But we were already persisting part of the state to the storage with `redux-persist` so i just extended it and added the ability to partially persist sub-elements. `redux-persist-transform-filter` helped a lot in this :)